### PR TITLE
Add shysank to maintainers

### PR DESCRIPTION
**Pull request by shysank**

### CODE REVIEW

**Comment by shysank** [comment]
cc @CecileRobertMichon

File: k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS [hunk 1]
```diff
@@ -6,11 +6,11 @@ approvers:
 - CecileRobertMichon
 - devigned
 - nader-ziada
+- shysank
 reviewers:
 - cpanato
 - juan-lee
 - mboersma
```
Comment (CecileRobertMichon) [inline k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS:13]: can you also please add @jackfrancis and @jsturtevant so this is up to date with our owners file?

Comment (shysank) [reply k8s.gcr.io/images/k8s-staging-cluster-api-azure/OWNERS:15]: done

**Comment by CecileRobertMichon** [comment]
/retest

check failure seems unrelated

**Comment by puerco** [comment]
CI failure fixed in https://github.com/kubernetes/k8s.io/pull/3495
/test pull-k8sio-verify

**Pull request merged**